state/logs.go

```diff
@@ -564,11 +591,20 @@ func (t *logTailer) tailOplog() error {
 			}
 			rec, err := logDocToRecord(t.modelUUID, doc)
 			if err != nil {
-				return errors.Annotate(err, "deserialization failed (possible DB corruption)")
+				if deserialisationFailures == 0 {
+					logger.Warningf("deserialization failed (possible DB corruption), %v", err)
```

Comment:
Should we be surfacing an error to the log channel that we have a bad record? It is a little interesting that we would be writing this to the controller log stream.
		return nil, errors.Annotate(err, "while parsing entity tag")
^ this error could be updated to give any sort of other context of the actual log message, etc. All we get told is "there exists a record that has a missing entity" but it doesn't include the other good fields for the log message for us to know what log record is bad.